index.js

```diff
@@ -31,32 +31,39 @@ Messer.prototype.fetchCurrentUser = function fetchCurrentUser() {
   return new Promise((resolve, reject) => {
     user.userID = this.api.getCurrentUserID()
 
-    this.api.getUserInfo(user.userID, (err, data) => {
-      if (err) return reject(err)
+    this.api.getUserInfo(user.userID, (userInfoError, userData) => {
+      if (userInfoError) {
+        reject(userInfoError)
+        return
```

Comment:
hey @Calvin-Huang, ive noticed you've removed the `return resolve...` pattern; whats the reasoning behind this?